html/user/per_app_list.php

```diff
@@ -63,13 +67,20 @@ function show_header($is_team, $apps, $appid, $is_total) {
 // show a user or team, with their credit for each app
 //
 function show_row($item, $apps, $is_team, $i) {
```

Comment:
Ok, so I will look into implementing full caching. Are there any suggestions or ideas on how to best do so?